### PR TITLE
Update bert.rst

### PR DESCRIPTION
**Pull request by jyang-aws**
Update LAMB and AMP tutorials (2.7 release)
This ties to PR from https://github.com/aws-neuron/aws-neuron-samples-staging/pull/41.

### CODE REVIEW

File: frameworks/torch/torch-neuronx/tutorials/training/bert.rst [hunk 1]
```diff
@@ -329,6 +330,47 @@ the BERT pretraining demo:
...
+   --grad_accum_usteps 128 |& tee run_pretrain_log.txt
+   
+Phase 1 BERT-Large pretraining with Automatic Mxed Precison (AMP)
+-----------------------------------------------
+Besides the `BFloat16 and stochastic rounding in phase 1 <https://github.com/aws-neuron/aws-neuron-sdk/edit/lamb-and-amp-tutorial-update/frameworks/torch/torch-neuronx/tutorials/training/bert.rst#id33>`__, you can also use AMP. The detailed background is at https://pytorch.org/docs/stable/amp.html. It uses both data type BFloat16 and Float32. With this setting, the loss ususally matches to Float32, but with a higher throughput. In applications sensitive to accuracy/loss values, AMP may be prefered over BFloat16 and Stochastic Roundings. A detailed comparison is available at `benchmark <https://awsdocs-neuron.readthedocs-hosted.com/en/latest/general/benchmarks/trn1/trn1-performance.html?highlight=performance#training-performance>`__.
+To launch the AMP, one additional command-line argument is needed ``--enable_pt_autocast``.
+
+.. code:: bash
+
+   cd ~/examples/dp_bert_hf_pretrain
```
Comment (jeffhataws) [inline frameworks/torch/torch-neuronx/tutorials/training/bert.rst:368]: Please add 

NEURON_STOCHASTIC_ROUNDING_EN=1

**Review by jeffhataws — CHANGES_REQUESTED** [review]

File: frameworks/torch/torch-neuronx/tutorials/training/bert.rst (hunk 1 above)
Comment (jyang-aws) [reply frameworks/torch/torch-neuronx/tutorials/training/bert.rst:368]: Thanks. added.

Comment (jeffhataws) [reply frameworks/torch/torch-neuronx/tutorials/training/bert.rst:368]: Sorry Jie, I meant add it to the AMP script NEURON_RT_STOCHASTIC_ROUNDING_EN=1. 

Also, I gave the wrong envvar name. Should have "_RT" in the name, like NEURON_RT_STOCHASTIC_ROUNDING_EN.

I don't know if it is needed for LAMP.

**Review by jeffhataws — CHANGES_REQUESTED** [review]

File: frameworks/torch/torch-neuronx/tutorials/training/bert.rst (hunk 1 above)
Comment (jyang-aws) [reply frameworks/torch/torch-neuronx/tutorials/training/bert.rst:368]: okay, I think once the BF16 flag is used, NEURON_RT_STOCHASTIC_ROUNDING_EN will be turn on automatically.

File: frameworks/torch/torch-neuronx/tutorials/training/bert.rst [hunk 2]
```diff
@@ -329,6 +330,47 @@ the BERT pretraining demo:
    to execute ``sudo rmmod neuron; sudo modprobe neuron`` in order to
    reload/reset the Neuron driver.
 
+Phase 1 BERT-Large pretraining with Layer-wise Adaptive Moments optimizer for Batch training (LAMB)
+-----------------------------------------------
```
Comment (jeffhataws) [inline frameworks/torch/torch-neuronx/tutorials/training/bert.rst:334]: This line needs to match the previous line in number of characters.

**Pull request closed without merging**